### PR TITLE
Script updates

### PR DESCRIPTION
**Pull request by suorcd**
Updating for consistency, a little bit of error check, and updating to meet shellcheck standards.

### CODE REVIEW

File: scripts/backup_repos.sh [hunk 1]
```diff
@@ -1,17 +1,21 @@
...
-cd $1/podverse-rn
+cd "$1/podverse-rn" || eixt 2
 git pull origin develop
 
-cd $1/podverse-web
+cd "$1/podverse-web" || eixt 2
 git pull origin develop
 
-cd $1/v4v-wallet-browser-extension
+cd "$1/v4v-wallet-browser-extension" || eixt 2
```
Comment (mitchdowney) [inline scripts/backup_repos.sh:20]: is this supposed to be "eixt 2"?

**Review by mitchdowney — CHANGES_REQUESTED** [review]
@suorcd thanks man! Sorry for the delay on seeing this. Still catching up on Github notifications.

I just have the one comment about "eixt 2"

File: scripts/backup_repos.sh (hunk 1 above)
Comment (suorcd) [reply scripts/backup_repos.sh:20]: nope that was a typo, should be exit. thanks copy/paste

on the exit code side, the numbers don't really matter. I was just trying to give a way to see the type
Though looking at https://tldp.org/LDP/abs/html/exitcodes.html code 2 has a bit more definition 'Misuse of shell builtins (according to Bash documentation)'

**Review by mitchdowney — APPROVED** [review]
Great. Thanks again!

**Pull request merged**